lib/constants.dart

```diff
@@ -3,6 +3,7 @@ library Constants;
 // Database Collections
 class DBCollections {
   static const String PRODUCTS = "products";
+  static const String USERS = 'users';
```

Comment:
I updated the rest of the code in my PR https://github.com/GospelAid/badiup/pull/2

I think we should follow the Dart guidelines about naming const parameters. https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
Okay, let's use `lowerCamelCase` for constants.